src/components/views/onboarding/styles.tsx

```diff
@@ -4,8 +4,7 @@ import {text, colors, shadows} from 'theme';
 
 export const styles = StyleSheet.create({
   title: {
-    ...text.largeBold,
-    marginBottom: 20
+    ...text.largeBold
```

Comment:
This applies to other screens as well (permissions?). Is that correct?

Reply:
I will check. maybe we should not have these shared styles...